apps/camera_pipe/Makefile

```diff
@@ -2,6 +2,8 @@ include ../support/Makefile.inc
 
 all: process
 
+CAMERA_ITERATIONS=5
```

Comment:
Doesn't this need to be CAMERA_ITERATIONS ?= 5? Otherwise the environment variable setting will get replaced?